examples/jsm/renderers/webgpu/WebGPUBackend.js

```diff
@@ -1065,15 +1065,14 @@ class WebGPUBackend extends Backend {
 
 	}
 
-	hasFeature( name ) {
+	hasFeature( name, silentError ) {
 
 		const adapter = this.adapter || _staticAdapter;
 
 		//
-
 		const features = Object.values( GPUFeatureName );
 
-		if ( features.includes( name ) === false ) {
+		if ( ! silentError && features.includes( name ) === false ) {
```

Comment:
I don't think we need to have an exception in `hasFeature()`, I think it would be simpler for the user to just receive a boolean if the feature does not exist in the current backend.

Reply:
Unrelated but do you have any idea on why the PostProcessing is not working with the WebGLBackend? If using any extra Pass the screen is completely black.

Reply:
Applied the feedbacks and remove the exception.
The exception can still be handled outside the backends if necessary since `hasFeature()` returns a boolean.

Reply:
> Unrelated but do you have any idea on why the PostProcessing is not working with the WebGLBackend? If using any extra Pass the screen is completely black.

I assume it's some global state that hasn't been updated during the render tree (`render()` over `render()`), just because WebGPU doesn't need to set global states like WebGL, but I'm not sure...